src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerDefaults.cs

```diff
@@ -44,7 +44,7 @@ public static class OpenIdConnectServerDefaults {
         /// <summary>
         /// Default value for <see cref="OpenIdConnectServerOptions.ValidationEndpointPath"/>.
         /// </summary>
-        public const string ValidationEndpointPath = "/connect/token_validation";
+        public const string ValidationEndpointPath = "/connect/introspect";
```

Comment:
I just followed the spec's non-normative examples for path naming... I suppose it doesn't matter much the path though...